Source/ASVideoNode.mm

```diff
@@ -707,6 +710,15 @@ - (void)resetToPlaceholder
 
 #pragma mark - Playback observers
 
+- (void)applicationDidBecomeActive:(NSNotification *)notification
+{
+  ASDN::MutexLocker l(__instanceLock__);
+  
+  if (_shouldBePlaying && ASInterfaceStateIncludesVisible(self.interfaceState)) {
```

Comment:
Other places in ASVideoNode seem to behave the same is why I did it this way, happy to remove it.